server/liveblog/blogs/blogs.py

```diff
@@ -15,8 +15,9 @@
     'guid': metadata_schema['guid'],
     'title': metadata_schema['headline'],
     'description': metadata_schema['description'],
-    'language': Resource.rel('languages', True),
-    'theme': Resource.rel('themes', True),
+    'theme': {
```

Comment:
This is to ensure that we freeze everything that is related to a blog when we publish it.
If someone change a theme file, it won't impact published blogs with the same theme.